stable/sonatype-nexus/templates/deployment.yaml

```diff
@@ -60,6 +60,9 @@ spec:
               name: nexus-data
             - mountPath: /nexus-data/backup
               name: nexus-backup
+            {{- with .Values.deployment.additionalVolumesMounts }}
```

Comment:
Why not map the configmap when config.enabled=true? That way you don't need to also set deployment.additionalVolumes and deployment.additionalVolumeMounts.

Reply:
You are right. I've updated the code accordingly.